src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Form/customizations_form_theme.html.twig

```diff
@@ -0,0 +1,39 @@
+{%- block customizations_widget -%}
+  <div class="col-md-12">
+    <h2>{{ 'Customization'|trans({}, 'Admin.Catalog.Feature') }}</h2>
+    <ul class="list-unstyled" id="customization-fields-list" data-prototype="{{ form_row(form.customization_fields.vars.prototype)|e('html_attr') }}">
+      {{ form_row(form.customization_fields) }}
+    </ul>
+    <button type="button" class="btn btn-outline-secondary" id="add-customization-btn">
+      <i class="material-icons">add_circle</i>
+      {{ 'Add a customization field'|trans({}, 'Admin.Catalog.Feature') }}
+    </button>
+  </div>
+{%- endblock customizations_widget -%}
+
+{%- block customization_field_row -%}
+  <li class="form-group row customization-field-item">
+    {{ form_widget(form.id) }}
+    <div class="col-md-5">
+      {{ form_widget(form.name) }}
+      {{ form_errors(form.name) }}
+    </div>
+    <div class="col-md-4">
+      {{ form_widget(form.type) }}
+      {{ form_errors(form.type) }}
+    </div>
+    <div class="col-md-1">
+      <button type="button" class="btn text-secondary remove-customization-btn">
```

Comment:
You should use the button from the FormType here not create it manually If what blocked you was the icon you can copy this type from my PR https://github.com/PrestaShop/PrestaShop/blob/9d791f206d896c4869f05fba96336077bdb47351/src/PrestaShopBundle/Form/Admin/Type/IconButtonType.php

(don't forget the associated block in the general theme)